test/sagemaker_tests/tensorflow/tensorflow1_training/integration/conftest.py

```diff
@@ -84,7 +85,10 @@ def region(request):
 def framework_version(request):
     return request.config.getoption('--framework-version')
 
-
+@pytest.fixture(scope='session', name='multi_region_support')
```

Comment:
Please return `None` for the fixture `n_virginia_sagemaker_session` and `n_virginia_ecr_image` if `not multi_region_support`, as an input guard. This will ensure that regions in other partitions do not fail, looking for access to `us-east-1`.

Reply:
Ack